queries.sql

```diff
@@ -7,3 +7,12 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pickachu';
 SELECT name, escape_attempts from animals where weight_kg > 10.5;
 SELECT * from animals where neutered = true;
 SELECT * from animals where weight_kg between 10.4 and 17.3;
+
+/* Second Excercise Queries */
+
```

Comment:
**Changes Required**
You're missing some queries
- [ ] UPDATE the animals table by setting the species column to unspecified, then roll back the change.
- [ ] `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- [ ] `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
- [ ] Commit the transaction.
- [ ] Inside a transaction delete all records in the animals table, then roll back the transaction.
- [ ] `DELETE` all animals born after Jan 1st, 2022
- [ ] Create a savepoint for the transaction
- [ ] `UPDATE` all animals' weight to be their weight multiplied by -1.
- [ ] * Rollback to the savepoint
- [ ] `UPDATE` all animals' weights that are negative to be their weight multiplied by -1.
- [ ] Commit transaction